searchlib/src/vespa/searchlib/common/bitvector.cpp

```diff
@@ -178,8 +178,17 @@ void
 BitVector::orWith(const BitVector & right)
 {
     verifyContains(*this, right);
-    IAccelrated::getAccelrator().orBit(getActiveStart(), right.getWordIndex(getStartIndex()), getActiveBytes());
 
+    if (right.size() < size()) {
```

Comment:
This should check if the right one ends before this one, since the right one could start before this one and also end before this one while still having greater size (should also add unit test for this case).

Reply:
I don't think this is a problem, since size() includes the portion before getStartIndex().
getEndIndex() doesn't make sense, and should be removed.

Reply:
That was my conclusion after looking at the details. Removed getEndIndex.